home/ivp-planning/user-interface/hierarchy-panel.md

```diff
@@ -0,0 +1,65 @@
+# Hierarchy Panel
+
+Every object that is added to an iVP project will get an associated entry in the hierarchy panel during its creation. This panel contains a list of all machines, objects, [markups](../advanced-tools/markup-tool.md), images, [DXF imports](../advanced-tools/dxf-tool.md), [PDF imports](../getting-started/importing-pdfs.md) and [virtual cameras](../advanced-tools/virtual-cameras.md) and is used to organize them in a tree structure. 
+
+Via this panel it is possible to collect objects in [folders](#folders), to [rename](../machines/renaming-objects-and-folders.md) objects and folders, to toggle their [visibility](#visibility) or to lock them to prevent unwanted changes.
+
+![](../../../.gitbook/assets/hierarchy.jpg)
+
+## The elements of the panel:
+
+|                                                           |                                                                                                                                                   |
+| --------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![](../../../.gitbook/assets/Hierarchy_search.jpg)        | The search bar makes it possible to only display such machines or objects which names contain the entered text (the search is not case sensitive) |
+| ![](../../../.gitbook/assets/Hierarchy_folder_button.jpg) | Adds a new folder to the hierarchy                                                                                                                |
+| ![](../../../.gitbook/assets/Hierarchy_color.jpg)         | Visualisation of the [color](../machines/highlighting-objects.md) set for the object(s)                                                           |
+| ![](../../../.gitbook/assets/Hierarchy_lock.jpg)          | Locks or unlocks the currently selected object(s)                                                                                                 |
+| ![](../../../.gitbook/assets/Hierarchy_visibility.jpg)    | Toggles the visibility of the currently selected object(s)                                                                                        |
+
+## Folders
+
+Folders can be used to structure projects. They can only be accessed in the hierarchy panel, however locking or hiding them will affect all objects inside - also in the [2D- ](../user-interface/the-2d-panel.md)and [3D-panel](../user-interface/the-3d-panel.md).
+
+### Creating folders:
+
+To create a folder, **press the "new folder" button** in the top of the hierarchy panel and a new folder will be added.
+
+![](../../../.gitbook/assets/Hierarchy_new_folder.jpg)
+
+{% hint style="info" %}
+The position of the new folder in will be in the topmost layer of the hierarchy. It can then be sorted into/under other folders or objects. 
```

Comment:
"The position of the new folder in will be in the topmost layer of the hierarchy"

change to

"The new folder will be placed at the topmost position of the hierarchy."